tests/shared/src/test/scala/cats/tests/OpSuite.scala

```diff
@@ -42,6 +43,36 @@ class OpSuite extends CatsSuite {
     checkAll("Category[Op[Function1, *, *]]", SerializableTests.serializable(Category[Op[Function1, *, *]]))
   }
 
+  {
+    implicit val catsDataDecidableForOp: Decidable[Op[Function1, Int, *]] = Op.catsDataDecidableForOp[Function1, Int]
```

Comment:
Same question here about ambiguities. It would be good if the tests can verify that it can be disambiguated in practice without needing help.